includes/class-comment.php

```diff
@@ -33,6 +33,8 @@ public static function init() {
 		\add_action( 'update_option_activitypub_allow_likes', array( self::class, 'maybe_update_comment_counts' ), 10, 2 );
 		\add_action( 'update_option_activitypub_allow_reposts', array( self::class, 'maybe_update_comment_counts' ), 10, 2 );
 		\add_filter( 'pre_wp_update_comment_count_now', array( static::class, 'pre_wp_update_comment_count_now' ), 10, 3 );
+
+		\add_action( 'transition_comment_status', array( self::class, 'maybe_announce_interaction' ), 20, 3 );
```

Comment:
Could this be moved to `/scheduler/class-comment.php`? We already schedule Outbox items for comments there, this feels like a good fit.